Makefile

```diff
@@ -1,20 +1,45 @@
-.PHONY: deps docs test build deploy
+.PHONY: help .install-poetry update install docs tests build deploy
+
+.DEFAULT_GOAL := help
+
+IS_POETRY := $(shell pip freeze | grep "poetry==")
+IS_TWINE := $(shell pip freeze | grep "twine==")
+
+
+help:
+	@echo "Please use 'make <target>' where <target> is one of"
+	@echo ""
+	@echo "  help           outputs this helper"
+	@echo "  build          builds the project .whl with poetry"
+	@echo "  update         updates the dependencies in poetry.lock"
+	@echo "  install        installs the dependencies in the env"
+	@echo "  test           run all the tests and linting"
+	@echo "  deploy         deploys the project using twine (not recommended, only use if really needed)"
```

Comment:
Good idea to add a `help` target. Could you order the targets alphabetically to make it a bit more user friendly?